README.md

```diff
@@ -26,11 +26,11 @@ pip install model_bakery
 # models.py
 
 class Customer(models.Model):
-    happy = models.BooleanField()
+    enjoy_jards_macale = models.BooleanField()
```

Comment:
I loved the Brazilian reference but I think we should name after something more widely known, like `has_fidelity_card` or `verified`.

Reply:
By the way, I didn't know him and after seeing this PR I decided to give his music a try. It's just wonderful.

In case somebody else is wondering who the heck is Jards Macale: https://www.youtube.com/watch?v=-eVbn_RuEdU :)

Reply:
Honestly, my initial purpose is exactly to make people to look for his songs =]

IMHO, it's just a minor example of a boolean field working as a flag. I mean, we can add another boolean field to improve our examples to use more "universal" attributes, but I'd really like to keep this one as well if you're ok with that.

What do you think? It'd be just a minor brazillian easter egg in our docs =]

Reply:
I'm going to give the casting vote to @amureki. :hear_no_evil:

Reply:
I am totally fine with it, sounds like a nice easter egg, yeah. :)

Reply:
Yaaaaaaay!!!!